changes/ce/fix-10100.en.md

```diff
@@ -0,0 +1 @@
+Fix channel crash for slow clients with enhanced authentication.
```

Comment:
Nit: It would be nice explain the implications of the error from users' perspective.